src/components/PostList/PostList.jsx

```diff
@@ -1 +1,10 @@
-export const PostList = () => <>Put the list here</>;
+import './PostList.scss';
+import { PostInfo } from '../PostInfo/PostInfo';
+
+export const PostList = ({ posts }) => (
+  <div className="PostList">
+    {posts.map(post => (
+      <PostInfo post={post} />
```

Comment:
Consider adding a `key` prop to the `PostInfo` component inside the `map` function. This helps React identify which items have changed, are added, or are removed, and is necessary for optimal rendering performance. Typically, you would use a unique identifier from the `post` object, such as `post.id`.